style/auth_mobile.css

```
@@ -0,0 +1,16 @@
+@media (max-width:767px){
+    br{
+        display:none !important;
+    }
```

P1:
br태그가 어떻게 사용될지 모르니 적용될 태그의 타켓팅을 더 자세히 해주시면 important 문도 빼실 수 있고 의도치 않은 동작도 막을 수 있어서 더 좋을 것 같습니다.

```suggestion
    .classname br,
    .classname2 br {
        display:none !important;
    }
```